python/cugraph-nx/README.md

```diff
@@ -0,0 +1,15 @@
+# cugraph_nx
+
+## Description
+[RAPIDS](https://rapids.ai) cugraph-nx is an experimental [backend to NetworkX](https://networkx.org/documentation/stable/reference/classes/index.html#backends)
+with minimal dependencies (`networkx`, `cupy`, and `pylibcugraph`).
+It is a work in progress, and cugraph should typically be used instead.
```

Comment:
on second thought...
I think we should do whatever a NetworkX user would expect, since this is after all a plugin to NetworkX.  I'm assuming that means do NOT raise an experimental warning, but maybe it's worth further discussion.

Reply:
I still think we should drop the last sentence though :)

Reply:
I'm so glad you looked at the README! I had completely forgotten about it. Last sentence deleted, and generally improved the README (but still lots to do).